### ISSUE TITLE
Add Dark Mode Support

### ISSUE THREAD

**Issue by pythonnelson** [hc-oss/react-multi-select-component#727]
Hello,

First, I want to thank you for your work on the `react-multi-select-component` package. It has been incredibly useful for my Next.js project. However, I have encountered an issue with dark mode support.

**Current Behavior:**  
Currently, the `react-multi-select-component` package does not seem to support dark mode. When switching to dark mode, the component's UI does not adapt accordingly, leading to poor user experience due to mismatched colors and readability issues.

**Expected Behavior:**  
It would be great if the component could automatically adapt to dark mode based on the parent theme or if there were props or configuration options available to easily style the component for dark mode.

**Additional Context:**

- I am using Next.js for my project.
- The dark mode is implemented using CSS classes/tailwind classes in my project.

**Proposed Solution:**

- Add default dark mode styling that activates based on a common dark mode class or media query.
- Provide props or configuration options to customize the component's styling for dark mode.

Thank you for considering this request. Dark mode has become an essential feature for many applications, and support for it would greatly enhance the usability of this package.

Best regards,  
Isaac NSB Kargbo

**Comment by yukendhiran** [comment]
Yeah I tought its problem with daisyUI, I also Experiencing the same, Other multi select component wrapped in thier own state, so I can not controll it with formik, like I cant conditionaly reset, but with this react-multi-select-component, I can able to control the state via formik, but it doesnot follow daisy ui, Did you find the work around?
